ckanext/security/plugin.py

```diff
@@ -1,13 +1,20 @@
+import logging
+
 import ckan.plugins as plugins
 import ckan.plugins.toolkit as toolkit
 import ckan.logic.schema
+
 from ckanext.security.model import define_security_tables
 from ckanext.security import schema
+from ckanext.security.resource_upload_validator import validate_upload_type, validate_upload_presence
+
+log = logging.getLogger(__name__)
 
 class CkanSecurityPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IRoutes)
     plugins.implements(plugins.IBlueprint)
+    plugins.implements(plugins.IResourceController, inherit=True)
```

Comment:
Handy thing, that `inherit=True`, otherwise you have to implement the entire interface rather than just overriding the ones you need to.

Reply:
See https://docs.ckan.org/en/2.7/extensions/plugin-interfaces.html#ckan.plugins.interfaces.IResourceController if you're interested.